lib/ace/lib/dom.js

```diff
@@ -276,7 +276,7 @@ if (typeof document !== "undefined") {
         exports.HAS_CSS_TRANSFORMS = true;
     if (!useragent.isEdge && typeof div.style.animationName !== "undefined")
         exports.HAS_CSS_ANIMATION = true;
-    div = null;
+    div.remove();
```

Comment:
this div element is not added to the document, so removing it will not have any effect

Reply:
I cared about memory leak, but no necessity to remove ?

Reply:
Since it is not connected to the document setting variable to null frees it up for garbage collector. remove is equivalent to `if (this.parentNode) this.parentNode.removeChild(this)` so when there is no parentNode it's a noop

Reply:
OK! I close this